internal/backend/auth/authloginhttpsvc/config.go

```diff
@@ -28,20 +50,44 @@ type descopeConfig struct {
 	ManagementKey string `koanf:"management_key"`
 }
 
+func (c descopeConfig) Validate() error {
+	if !c.Enabled {
+		return nil
+	}
+
+	if c.ProjectID == "" {
+		return errors.New("project_id is required")
+	}
+
+	if c.ManagementKey == "" {
```

Comment:
AFAIK management key isn't required. For example, I have a Descope config in my local server that doesn't have it.

Reply:
removed mgmt key all togethrol.